tests/test_splicing.py

```diff
@@ -39,3 +39,64 @@ def test_splice(node_factory, bitcoind):
     # Check that the splice doesn't generate a unilateral close transaction
     time.sleep(5)
     assert l1.db_query("SELECT count(*) as c FROM channeltxs;")[0]['c'] == 0
+
+
+@pytest.mark.openchannel('v1')
+@pytest.mark.openchannel('v2')
+@unittest.skipIf(TEST_NETWORK != 'regtest', 'elementsd doesnt yet support PSBT features we need')
+def test_splice_out(node_factory, bitcoind):
+    l1, l2 = node_factory.line_graph(2, fundamount=1000000, wait_for_announce=True, opts={'experimental-splicing': None})
+
+    chan_id = l1.get_channel_id(l2)
+
+    # add extra sats to pay fee
+    funds_result = l1.rpc.recvpsbt(100000)
```

Comment:
You need to write a separate set of tests for recvpsbt (is that a good name?  addpsbtoutput?).  It must test the case where we create, and simply enhance, a PSBT.

Reply:
Wrote a test and put it in test_wallet.py that tests both these flows. Going to let CI chew on it while I brainstorm a better name 👀